local_setup.md

```diff
@@ -0,0 +1,193 @@
+# Local Setup
+
+For developing you want to set up the renuo-cms and the renuo-upload locally. They must be set up once and can then be used by every application that uses one or both of those services.
+
+## Step by Step Manual
+
+### Install renuo-cms-api
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-cms-api.git
+cd renuo-cms-api
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Install renuo-upload signing
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-upload-signing.git
+cd renuo-upload-signing
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Set Up a Amazon S3 bucket
+
+1.  Log in to amazon
+
+1.  Navigate to S3 by clicking on the S3 icon:
+
+   ![S3-Icon](s3-icon.png)
+
+1.  Create new bucket by clicking on blue button on the top with text "create bucket":
+
+    ![create bucket](create-bucket-button.png)
+
+1.  Name it however you like. We request to do something like ```renuo-upload-<yourname>-development``` and choose to place where your server should be. We usually use Frankfurt.
+
+      ![configure bucket](new-s3-bucket.png)
+
+1.  Click on your project and navigate to Properties (right side of navigation). ![](properties-button.png) Click on the Permission section for folding it out and click on the butten called "Add CORS Configuration".
+
+  Insert this:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+    <CORSRule>
+        <AllowedOrigin>*</AllowedOrigin>
+        <AllowedMethod>GET</AllowedMethod>
+        <AllowedMethod>POST</AllowedMethod>
+        <MaxAgeSeconds>3000</MaxAgeSeconds>
+        <AllowedHeader>*</AllowedHeader>
+    </CORSRule>
+</CORSConfiguration>
+```
+1.  Now click save, then close and after that the blue save button.
+
+### Set UP Amazon IAM
+
+1.  Log in to amazon if you aren't already because you just set up your s3 bucket.
+
+1.  Navigate to IAM by clicking on the "Identity & Access Management" icon:
+
+  ![IAM-icon](IAM-icon.png)
+
+1.  Navigate to Users in the navigation on the left side.
+
+  ![Users in Navigation](users-in navigation.png)
+
+1.  Click on the blue button with a white "create new users"-writing:
+
+  ![Create New Users](better-create-new-users-button.png)
+
+1.  Enter your name into the first field.
+
+  ![Insert Your Name](insert-your-name.png)
+
+1.  Press "create" which you'll find in the lower right corner. It's blue with white writing:
+
+  ![Create Button](better-create-button.png)
+
+1.  Click on "Show User Security Credentials" and you see your needed keys. **IMPORTANT: **Download and store them in a secure place.
```

Comment:
okey :) the problem with bold disturbes the flow of reading a bit if you don't want to change it, it's ok for me to leave it like this